### PR TITLE
Fix upgrade discord.py 2.x.x

### PR DESCRIPTION
**Pull request by neihousaigaai**

### CODE REVIEW

File: commands/admin.py [hunk 1]
```diff
@@ -247,11 +248,13 @@ async def test_admin_command(guild):
     # TEST add/remove user to/from channel
     await add_user_to_channel(guild, public_user, channel_name, is_read=True, is_send=True)
     await asyncio.sleep(2)
-    assert isinstance(discord.utils.get(channel.members, name=public_user.name), discord.Member)
+    member = discord.utils.get(channel.members, name=public_user.name)
```
Comment (nitro2) [inline commands/admin.py:251]: Need await?

File: commands/admin.py [hunk 2]
```diff
@@ -247,11 +248,13 @@ async def test_admin_command(guild):
...
+    assert isinstance(member, discord.Member)
     await asyncio.sleep(5)
     await remove_user_from_channel(guild, public_user, channel_name)
     await asyncio.sleep(5)
-    assert discord.utils.get(channel.members, name=public_user.name) is None
+    member = discord.utils.get(channel.members, name=public_user.name)
```
Comment (nitro2) [inline commands/admin.py:256]: Need await?

Comment (neihousaigaai) [reply commands/admin.py:256]: No need to add await here.

File: commands/admin.py (hunk 1 above)
Comment (neihousaigaai) [reply commands/admin.py:251]: No need to add await here.

**Review by huybt22 — APPROVED** [review]
Thank you for your effort!

**Pull request merged**